### DISCUSSION TITLE
Fl0

### DISCUSSION THREAD

**Discussion by sersebas08** [community/community#134847] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body
I'm trying to use fl0 but I can't register... any recommendations to fix this or any alternatives to use, thanks!!

**Comment by ebndev** [comment, marked as answer]
Hi @sersebas08, thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting to ensure discussions are adhering to our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md). 

We’ll be locking this post for now, but please feel encouraged to create a new discussion in English if you still need support.
